src/vm/stablehashcode.cpp

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "common.h"
+#include "stablehashcode.h"
+#include "typehashingalgorithms.h"
+
+int GetStableHashCode(TypeHandle type)
+{
```

Comment:
Could you please reconcile it with the hashcode computation added in earlier Fadi's checkin (check ZapImage::OutputTypesTableForReadyToRun in https://github.com/dotnet/coreclr/pull/3735)?